crates/api/src/layout/fixed_size.rs

```diff
@@ -194,3 +194,19 @@ impl Layout for FixedSizeLayout {
         self.desired_size.borrow().size()
     }
 }
+
+fn text(widget: &WidgetContainer) -> Option<String> {
+    if let Some(localizable) = widget.try_get::<bool>("localizable") {
+        if *localizable {
+            if let Some(localized_text) = widget.try_get::<String>("localized_text") {
+                return Some(localized_text.clone());
+            }
+        }
+    }
+
+    if let Some(text) = widget.try_get::<String>("text") {
+        return Some(text.clone());
+    }
+
+    None
+}
```

Comment:
I used to work with PHP and Java HTML template engines. Those used the string interpolation technique, like
`{#text}` String would be replaced by the value in a dictionary of the corresponding locale.
The locale or the `{#text}` property could be the key to the value in the dictionary.
This way we could get rid of the `localizable` property.

Reply:
I explicit do it not this way, that you can write you ui complete without localization in mind (in one language). If you want to add other languages you can register a localization service and use the normal text Strings as key.

I see this a start point and further ideas and contributions in this are welcome.

Reply:
i see...